rllib/examples/env/utils/mixins.py

```diff
@@ -27,10 +28,10 @@ def _reset_info(self):
 
     def _get_episode_info(self):
         return {
-            "CC": sum(self.cc_count) / len(self.cc_count),
-            "DD": sum(self.dd_count) / len(self.dd_count),
-            "CD": sum(self.cd_count) / len(self.cd_count),
-            "DC": sum(self.dc_count) / len(self.dc_count),
+            "CC": np.mean(self.cc_count),
+            "DD": np.mean(self.dd_count),
+            "CD": np.mean(self.cd_count),
+            "DC": np.mean(self.dc_count),
```

Comment:
At least here we should probably continue to work with primitives

```suggestion
            "CC": np.mean(self.cc_count).item(),
            "DD": np.mean(self.dd_count).item(),
            "CD": np.mean(self.cd_count).item(),
            "DC": np.mean(self.dc_count).item(),
```